bindings/python/dlite-entity.i

```diff
@@ -257,16 +257,18 @@ Instance(driver, location, options, id=None)
     and `options`. `id` is the id of the instance in the storage (not
     required if the storage only contains more one instance).
 
-Instance(uri, dimensions, properties, iri, description)
+Instance(uri, dimensions, properties, description)
```

Comment:
We should remember to update the documentation.

In this case we should say that Instance() is now intended to be called from one of the new class methods

    create_from_metaid(cls, metaid, dims, id=None)
    create_from_url(cls, url, metaid=None)
    create_from_storage(cls, storage, id=None, metaid=None)
    create_from_location(cls, driver, location, options=None, id=None)
    create_metadata(cls, uri, dimensions, properties, description)

and refer to their documentation for details.